cmd/lotus-shed/migrations.go

```diff
@@ -163,21 +164,7 @@ var migrationsCmd = &cli.Command{
 				return err
 			}
 
-			preMigration1Time := time.Since(startTime)
-
-			ts2, err := cs.GetTipsetByHeight(ctx, blk.Height-15, migrationTs, false)
-			if err != nil {
-				return err
-			}
-
-			startTime = time.Now()
-
-			err = preUpgradeActorsFunc(ctx, sm, cache, ts2.ParentState(), ts2.Height()-1, ts2)
-			if err != nil {
-				return err
-			}
-
-			preMigration2Time := time.Since(startTime)
+			preMigrationTime := time.Since(startTime)
```

Comment:
Does this mean we only need a single pre-migration now?

Reply:
Yessir -- this migration is easy-peasy, so a single pre-migration gets the job done.